src/components/Tabs/Tabs.jsx

```diff
@@ -45,6 +45,19 @@ class Tabs extends React.Component {
   }
 
   renderTab = (tab, index) => {
+    if (tab.renderType && tab.renderType === 'html') {
+      return (
+        <li
+          role="tab"
+          onClick={() => this.onClick(tab, index)}
+          onKeyDown={(event) => this.onKeyDown(event, tab, index)}
+          className={this.getTabClassNames(tab, index)}
+          key={index}
+          tabIndex="0"
```

Comment:
This looks good, but it'd be nice to get rid of a duplication of props that we pass to `<li>` element. Maybe move them to an object and spread in both places?

Reply:
Makes sense! Updated..